### PR TITLE
Made Quote Translatable

### PR DESCRIPTION
**Pull request by zo-shin**
Made the quote translatable

### CODE REVIEW

**Comment by y20k** [comment]
Hi @zo-shin 

the quote "Nothing is so perfect that it can't be complained about." was the motto of the podcast that made me understand what podcasting is about / what it can achieve: [Hypercritical (5by5)](https://5by5.tv/hypercritical).

I think there is not really a benefit in translating it. But there is a little benefit in leaving it English. Users, who are curious, can google the phrase and find out about the mentioned podcast.

**Comment by zo-shin** [comment]
I feel it's a little bit awkward to show the English quote on the start page. It should be understood by all the users. If you insist, I can close it.

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -157,6 +160,7 @@
     <string name="descr_expanded_player_previous_chapter_button">上一章</string>
     <string name="descr_expanded_player_replay_button">重复最后 10 秒</string>
     <string name="descr_expanded_player_seekbar">节目回放时间</string>
+    <string name="descr_expanded_player_shownotes_symbol">Show notes symbol显示注释符号</string>
```
Comment (comradekingu) [inline app/src/main/res/values-zh-rCN/strings.xml:163]:
```suggestion
    <string name="descr_expanded_player_shownotes_symbol">显示注释符号</string>
```
?

**Comment by comradekingu** [comment]
@zo-shin
I thought this was going to remove a "translatable false". Improvements to the translation can be done from
https://hosted.weblate.org/projects/escapepod/android-app/zh_Hans/

**Pull request merged**